CHANGELOG.md

```diff
@@ -1,18 +1,34 @@
-# Notebooks 0.10.0 (TBD)
+# Notebooks 0.11.0 (TBD)
```

Comment:
This doesn't look right. I'm assuming we don't start the 0.11 section in the 0.10 release?

Reply:
i followed the model i saw.  Maybe it didn't need the 0.11 during release.